jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -10,13 +10,78 @@
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import (PluginTemplateMixin, ViewerSelect, LayerSelect,
                                         PlotOptionsSyncState)
+from jdaviz.core.user_api import PluginUserApi
 from jdaviz.core.tools import ICON_DIR
 
 __all__ = ['PlotOptions']
 
 
 @tray_registry('g-plot-options', label="Plot Options")
 class PlotOptions(PluginTemplateMixin):
+    """
+    The Plot Options Plugin gives access to per-viewer and per-layer options and enables
+    setting across multiple viewers/layers simultaneously.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``multiselect``:
+      whether ``viewer`` and ``layer`` should both be in multiselect mode.
+    * ``viewer`` (:class:`~jdaviz.core.template_mixin.ViewerSelect`):
+    * ``layer`` (:class:`~jdaviz.core.template_mixin.LayerSelect`):
+    * :meth:`select_all`
+    * ``layer_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)
+    * ``subset_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)
+    * ``show_axes`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)
+    * ``collapse_function`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      only avilable for Cubeviz
+    * ``line_color`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Imviz
+    * ``line_width`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Imviz
+    * ``line_opacity`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Imviz
+    * ``as_steps`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Imviz
+    * ``uncertainty`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Imviz
+    * ``stretch`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Specviz
+    * ``stretch_perc`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Specviz
+    * ``stretch_min`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Specviz
+    * ``stretch_max`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Specviz
+    * ``bitmap_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Specviz
+    * ``color_mode`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Specviz
+    * ``bitmap_color`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):

```

Comment:
This is only for when it is Monochromatic, right?
```suggestion
    * ``color`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
```

Reply:
yes, but shouldn't be confused with line color.

Reply:
OK, I updated my suggestion above though the words might need polishing.